apps/sms/views/shared/test/unit/contacts_test.js

```diff
@@ -756,29 +609,22 @@ suite('Contacts', function(done) {
         // contacts were found
         assert.ok(Array.isArray(contacts));
         assert.equal(contacts.length, 1);
-
-        // navigator.mozContacts.find was called?
-        assert.equal(mozContacts.mHistory.length, 1);
-        assert.deepEqual(mozContacts.mHistory[0].filter, filter);
       }).then(done, done);
```

Comment:
Yeah! Even i dont get why it worked before, but after using sinon sandbox, it works only if i define a stub, or test gives error that "navigator.mozContacts.find" is not defined.